api/create_distributed_restore_point.md

```diff
@@ -0,0 +1,57 @@
+# create_distributed_restore_point()
+Creates a same-named marker record, for example `restore point`, in the
+write-ahead logs of all nodes in a multi-node TimescaleDB cluster.
+
+The restore point can be used as a recovery target on each node, ensuring the
+entire multi-node cluster can be restored to a consistent state. The function
+returns the write-ahead log locations for all nodes where the marker record was
+written.
+
+This function is similar to the PostgreSQL function
+[`pg_create_restore_point`][pg-create-restore-point], but it has been modified
+to work with a distributed database.
+
+This function can only be run on the access node, and requires superuser
+privileges.
+
+## Required arguments
+
+|Name|Description|
+|-|-|
+|`name`|The restore point name|
+
+## Returns
+
+|Column|Type|Description|
+|-|-|-|
+|`node_name`|NAME|Node name, or `NULL` for access node|
+|`node_type`|TEXT|Node type name: `access_node` or `data_node`|
+|`restore_point`|[PG_LSN][pg-lsn])|Restore point log sequence number|
```

Comment:
I guess we don't need that extra bracket here:  [PG_LSN][pg-lsn]`)`

Reply:
```suggestion
|`restore_point`|[PG_LSN][pg-lsn]|Restore point log sequence number|
```


Good spot!